retrace/glstate_shaders.cpp

```diff
@@ -1138,6 +1138,28 @@ void dumpShadersStorageBufferBlocks(StateWriter &writer, Context &context,
                 "GL_BUFFER_DATA_SIZE",
                 getProgramResourcei(program, GL_SHADER_STORAGE_BLOCK,
                                     ssbbResourceIndex, GL_BUFFER_DATA_SIZE));
+
+            auto outputIfReferenced = [&writer,ssbbResourceIndex, program](
```

Comment:
Instead of the lambda, please just use a static const table of GLenum, and a for cycle.